SQF/dayz_code/system/zombie_agent.fsm

```diff
@@ -700,7 +701,7 @@ class FSM
   initState="init";
   finalStates[] =
   {
-    "End"
+    "End",
```

Comment:
Never tested this in arma ( in php this works). BUt this doesn't create syntax error?